### PR TITLE
'Set Up a Hello Microverse Project'

### PR DESCRIPTION
**Pull request by zunairkhan811**
The project has been made according to the following requirement:
1-Linters Added
2-test.md file and node-modules ignored by using .gitignore file
3-HTML file has been made saying hello to microverse
4- .css file has been made and the 'favorite-color' class has been given to the h1 heading in the HTML file.
5- Readme file has been added according to the professional readme-style template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Just Hello to Microverse</title>
+</head>
+<body>
+    <h1>Hello to Microverse</h1>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [x] As stated the project requirement Kindly make sure that A .css file that includes `one class` with styles for your header (make it in your favorite color).
- Capture

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author**
+
+- GitHub: [@zunairkhan811](https://github.com/zunairkhan811)
```
Comment (Abenezer-Tilahun) [inline README.md]: - [x] Kindly I recommend you to update your README file, README file should be descriptive and **All main headers** from the [README template](https://github.com/microverseinc/readme-template) should be present in a yours repo and Please Make sure the you has added the **section Future Features**, including 1-3 potential features to be added..

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @zunairkhan811   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub-Flow.
✔️ Good README
✔️ Good commit messages

almost

## Required Changes ♻️

- [x] Kindly Make sure that your Pull Request should have a descriptive title telling you what changes are inside Just like **"Set up a Hello Microverse project"** instead of "Add Linters" and Pull Request should have a short summary describing introduced changes in more detail.

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @zunairkhan811 👋 👋

Good job done so far! 🥇


Your project is complete! There is nothing else to say other than... it's time to merge it 🦾
Congratulations! 🎉

Highlights
✔️Made the required changes👍
✔️professional README👍
✔️No linter errors👍

Optional requirements
[optional] **Please kindly ensure to add other sections in this [template](https://github.com/microverseinc/readme-template) to your README file, this will make your repo to be more professional. Thanks. 👍** 

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**